sys-devel/automake/automake-1.15.1-r1.ebuild

```diff
@@ -0,0 +1,109 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit eutils versionator
+
+case ${PV} in
+9999)
+	EGIT_REPO_URI="git://git.savannah.gnu.org/${PN}.git
+		http://git.savannah.gnu.org/r/${PN}.git"
+
+	inherit git-r3
+	;;
+*)
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~ppc-aix ~x64-cygwin ~amd64-fbsd ~sparc-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~m68k-mint ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
+	if [[ ${PV/_beta} == ${PV} ]]; then
+		MY_P=${P}
+		SRC_URI="mirror://gnu/${PN}/${P}.tar.xz
+			ftp://alpha.gnu.org/pub/gnu/${PN}/${MY_P}.tar.xz"
+	else
+		MY_PV="$(get_major_version).$(($(get_version_component_range 2)-1))b"
+		MY_P="${PN}-${MY_PV}"
+
+		# Alpha/beta releases are not distributed on the usual mirrors.
+		SRC_URI="ftp://alpha.gnu.org/pub/gnu/${PN}/${MY_P}.tar.xz"
+	fi
+	S="${WORKDIR}/${MY_P}"
+	;;
+esac
+
+DESCRIPTION="Used to generate Makefile.in from Makefile.am"
+HOMEPAGE="https://www.gnu.org/software/automake/"
+
+LICENSE="GPL-2"
+# Use Gentoo versioning for slotting.
+SLOT="${PV:0:4}"
+IUSE=""
+
+RDEPEND="dev-lang/perl
+	>=sys-devel/automake-wrapper-10
+	>=sys-devel/autoconf-2.69
+	sys-devel/gnuconfig"
+DEPEND="${RDEPEND}
+	sys-apps/help2man"
+
+src_prepare() {
+	default
+	export WANT_AUTOCONF=2.5
+	# Don't try wrapping the autotools this thing runs as it tends
+	# to be a bit esoteric, and the script does `set -e` itself.
+	./bootstrap || die
+	sed -i -e "/APIVERSION=/s:=.*:=${SLOT}:" configure || die
+}
+
+src_configure() {
+	econf --docdir="\$(datarootdir)/doc/${PF}"
+}
+
+src_test() {
+	emake check
```

Comment:
https://devmanual.gentoo.org/ebuild-writing/eapi/ doesn't say so, so I dug a little deeper, and https://devmanual.gentoo.org/ebuild-writing/functions/src_test/index.html does say so.
tl;dr: 👍